lib/dependencies/JsonExportsDependency.js

```diff
@@ -74,22 +77,20 @@ class JsonExportsDependency extends NullDependency {
 	 */
 	updateHash(hash, context) {
 		if (this._hashUpdate === undefined) {
-			this._hashUpdate = this.exports
-				? JSON.stringify(this.exports)
-				: "undefined";
+			this._hashUpdate = this.data ? JSON.stringify(this.data) : "undefined";
 		}
 		hash.update(this._hashUpdate);
```

Comment:
```
this.data.updateHash(hash);
```

Reply:
This can be added to `JsonData` (if not there yet) to do hashing based on the buffer version

Reply:
fixed